p2p/proto/transaction.proto

```diff
@@ -70,6 +71,7 @@ message Transaction
         Felt252          nonce            = 4;
         Felt252          address_salt     = 5;
         repeated         Felt252 calldata = 6;
+        Address          address          = 7;
     }
```

Comment:
I don't think that we need an `address` field in DEPLOY_ACCOUNT, same for v3 (the address of the would-be account is determined by the other fields). The JRPC doesn't have an address.

Reply:
You're right, in case of v3 it's not necessary. From our perspective because it does not take part in transaction hash calculation.

Reply:
In case of v1 we do need it for transaction hash calculation but we can compute it :+1: .